src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs

```diff
@@ -227,6 +271,14 @@ private async Task<(string sourceBranch, string sourceVersion)> GetSourceBranchI
                 return (null, null);
             }
 
+            var oldestSupportedArcadeSDKDate = new DateTimeOffset(2020, 01, 28, 0, 0, 0, new TimeSpan(0, 0, 0));
```

Comment:
You might include a message here saying you can specify the arcade version to use.  Surprisingly, I managed to publish a build with an arcade version from last July by explicitly passing the parameters.

Reply:
Yeah, definitely. Totally missed that!